src/uikit.cs

```diff
@@ -14523,6 +14786,9 @@ public interface UIPrintInteractionControllerDelegate {
 #endif
 		nfloat CutLengthForPaper (UIPrintInteractionController printInteractionController, UIPrintPaper paper);
 
+		[iOS (9, 0)]
+		[Export ("printInteractionController:chooseCutterBehavior:"), DefaultValue ("UIPrinterCutterBehavior.NoCut"), DelegateName ("UIPrintInteractionCutterBehavior")]
+		UIPrinterCutterBehavior ChooseCutterBehavior (UIPrintInteractionController printInteractionController, NSNumber [] availableBehaviors);
```

Comment:
Yes, but the reason we wanted it to be executed is because we did not know the type of `availableBehaviors` array

    - (UIPrinterCutterBehavior) printInteractionController:(UIPrintInteractionController *)printInteractionController
    chooseCutterBehavior:(NSArray *)availableBehaviors NS_AVAILABLE_IOS(9_0);`

 but now thanks to [swift docs](https://developer.apple.com/reference/uikit/uiprintinteractioncontrollerdelegate/1618178-printinteractioncontroller) we know what it is:

> An array of NSNumber objects identifying the printer’s available cutter behaviors. Each number corresponds to one of the constants defined in UIPrinterCutterBehavior.